src/Components/RestaurantButton.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+const RestaurantButton = ({ clickAction }) => {
+  return (
+    <button onClick={clickAction} className="btn btn-primary">
```

Comment:
Nice one! I think it would be more intuitive to call the prop with the actual function name - `orderOne` instead of `clickAction`.

Reply:
done